Changes

```diff
@@ -36,6 +36,10 @@ Next version (4.05.0):
    and Damien Doligez, discussion with Alain Frisch and Xavier Leroy,
    feature request from the Coq team)
 
+- PR#7172, GPR#970: add extra (ocamlc -config) options
+  int_size, word_size, ext_exe
+  (Gabriel Scherer, request by Daniel Buenzli)
```

Comment:
Well, getting useful commits stuck in an ambitious pull request is bad. I can cherry-pick and go from there -- I just did a pass on the Changelog.